apps/src/sites/studio/pages/home/courses.js

```diff
@@ -0,0 +1,26 @@
+import $ from 'jquery';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Courses from '@cdo/apps/templates/teacherHomepage/Courses';
+import {initCourseExplorer} from '@cdo/apps/courseExplorer/courseExplorer';
+
+$(document).ready(showCourses);
+
+function showCourses() {
+  // Initialize the non-React Course/Tool Explorer component code.
+  initCourseExplorer();
+
+  const coursesData = document.querySelector('script[data-courses]');
+  const configCourses = JSON.parse(coursesData.dataset.courses);
+
+  const englishTeacherData = document.querySelector('script[data-englishteacher]');
+  const configEnglishTeacher = JSON.parse(englishTeacherData.dataset.englishteacher);
+
+  ReactDOM.render (
+    <Courses
+      courses={configCourses}
+      englishTeacher={configEnglishTeacher}
```

Comment:
I wonder if we'd be better off calling this something like isEnglishTeacher to make it clearer it's a bool.